apps/test/util/runTests.js

```diff
@@ -70,6 +70,7 @@ exec(command, function (err, stdout, stderr) {
   mochify(globs.join(' '), {
     grep: grep,
     debug: process.env.mocha_debug,
+    extension: ['.jsx'],
```

Comment:
However, the mochify API option is `extension` (singular) to match the long-form CLI option, [as more-or-less documented here](https://github.com/mantoni/mochify.js#api).